generate/layout.go

```diff
@@ -211,6 +211,7 @@ var layout = apiInfo{
 		"releaseHostReservation",
 		"updateHost",
 		"updateHostPassword",
+		"migrateSecondaryStorageData",
```

Comment:
Do you think this would be better to be placed in the ImageStoreService group?

Reply:
Hi, I'm just following the api refrence https://cloudstack.apache.org/api/apidocs-4.19/ where this is under Host, but yes I believe it's better placed in the ImageStoreService.

Reply:
Ah, got it. My bad. I guess it would make sense to correct it there as well (eventually).